### ISSUE TITLE
荣誉统计错误的问题

### ISSUE THREAD

**Issue by kongccc** [slowtorta/turtlewow-bug-tracker#5123]
GM 您好，我是一名乌龟服忠实玩家，向您反映一个关于荣誉统计错误的问题。我的账号 fengchen3669 角色名 stefan 血环服务器，部落盗贼。10 月25 日至11月 1日，这个 CD 期间，我荣誉打了 32 万多，11 月1日周三结算的时候，显示我上周荣誉为 161241，少计算 16 万多荣誉(以上情况都有截图)。11 月1 日，我在游戏内向 GM 反映荣誉有错误的问题，GM 在游戏内通过邮件反馈表示，“服务器更新时没下线，下次更新会补的。”11月2日,服务器更新后,为我补登了164868荣誉，但随后服务器重启，补登的荣誉又被扣除。一周打 16 万多荣誉，平均每天在线打战场 10 小时，作为一名个人玩家来讲,付出了大量心血,非常不容易恳请 GM 能给我补登 16 万荣誉，万分感谢，衷心祝愿乌龟服越办越好。以上情况我都有截图，需要的话可以提供。

**Comment by Muigin** [comment]
Please submit the issues in English. We're unable to accurately translate bug reports submitted in Chinese through automatic translation, which makes it difficult for us to understand the exact nature of the issue. Thank you!

**Comment by kongccc** [comment]
GM Hello, I am a loyal player in turtle suit, to reflect to you on the honor of a statistical error. My account fengchen3669 character name stefan Blood Ring Server, tribal thieves. Between October 25th and November 1st, during this CD period, I hit over 320,000 with honors. When I settled on Wednesday, November 1st, it showed that I had 161241 honors last week, less than 160,000 honors (screenshots of all the above) . On November 1, I told GM in-game that there was a problem with honor. GM emailed me back in-game saying, “The server wasn't offline when it was updated. I'll make it up next time.” On November 2, after the server was updated, I was awarded 164868 Honours, which were later deducted when the server was rebooted. Playing more than 160,000 honors a week, and playing online for an average of 10 hours a day. As an individual player, I have put in a lot of effort, and it is not easy to ask GM to add 160,000 honors to my account. Thank you very much, i sincerely wish the turtle suit better and better. I have screenshots of all of the above and can provide them if needed.
